py/server/tests/test_udf_args.py

```diff
@@ -514,6 +521,5 @@ def my_sum_error(p1: np.int32, *args: np.int16) -> int:
         self.assertRegex(str(cm.exception), "my_sum_error: Expected argument .* got int")
 
 
-
 if __name__ == "__main__":
     unittest.main()
```

Comment:
I don't see any tests for cases like these:
1. Arg `Optional[float, np.float32, np.float64]` and all permutations of the type list should always pass in a `float`.
2. Arg `Optional[np.float32, np.float64]` and all permuations of the type list should always(?) pass in a `np.float64` since it is a faster conversion based on my benchmarks.
3. etc